CONTRIBUTING.adoc

```diff
@@ -0,0 +1,90 @@
+
+2. Provide as much information about your environment, software version, and relevant dependencies as possible. For example, let us know what version of kubernetes you are using, which launch type for the kubernetes cluster (e.g. EKS or kops), and the environment your code is running in. (e.g Cloud9).
+
+If, after doing the above steps, you determine that you need to submit a bug report, refer to the next section.
+
+=== Submitting a Bug Report
+So that we are able to assist you as effectively as possible with the issue, please ensure that your bug report has the following:
+
+* A short, descriptive title. Ideally, other community members should be able to get a good idea of the issue just from reading the title.
+* A detailed description of the problem you're experiencing. This should include:
+  * Expected behavior of the workshop and the actual behavior exhibited.
+  * Any details of your application environment that may be relevant.
+  * Commands and output used to reproduce the issue.
+* [Markdown][markdown] formatting as appropriate to make the report easier to read; for example use code blocks when pasting a code snippet and exception stacktraces.
+
+=== Tracking your Bug Report
+You will automatically be notified of any changes to a bug report. Either a maintainer or a community member may comment on the bug report requiring further input. We encourage all contributors to stay active on their bug reports reports. This helps the community to focus on issues with more influence and activity.
+
+We understand that contributors cannot devote 100% of their time to a project, just as our maintainers cannot devote 100% of their time to the project.. We want to squash bugs as often as we can, but sometimes require comments from the submitter or community in order to proceed.
+
+If a bug report has been inactive (waiting on comment from submitter or community) for 14 days, a maintainer may label the issue as stale.  Once a bug report has been inactive for more than 28 days, the issue will be administratively closed. Issues can be re-opened after being administratively closed by commenting on the issue requesting to re-open and providing the necessary comment.
+
+== Enhancements
+Like bug reports, enhancements are submitted through the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page.
+
+As with Bug Reports, please do a search of the open requests first before submitting a new one to avoid duplicates. If you find an existing one, give it a +1.
+
+[NOTE]
+If this is an enhancement you intend to implement, please be sure to submit the enhancement *before* working on any code changes. This will allow members on the workshop team to have a discussion with you to ensure that it's the right design and that it makes sense to include in the workshop.
+
+Enhancements are labeled with link:https://github.com/aws-samples/aws-workshop-for-kubernetes/labels/Enhancement[Enhancement].
```

Comment:
Are we going to retrofit existing issues with this label?

Reply:
The manual way. Working on that over the next few days.